bundles/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/internal/handler/SystemInfoHandler.java

```diff
@@ -229,26 +226,22 @@ private boolean updateProperties() {
      * @return true if channel groups where added
      */
     private boolean addDynamicChannels() {
```

Comment:
Please also update the javadoc of the method.

Reply:
The method still do the same. Miss i something?

Reply:
The thing UUID is different and mentioned in the javadoc.

Reply:
I now noticed you removed that part in the javadoc completely. I don't agree with removing the last part. If the thing type changes (when dynamic channel groups are added), there will be re new thing created and a new initialization, so the return early comment is still valid.

Reply:
Done